script-docs/hub/ui/annotate/modify_test_cases.rst

```diff
@@ -0,0 +1,206 @@
+
+.. tip::
+
+   Test cases (conversations) are part of datasets. For information on creating and managing datasets, see :doc:`/hub/ui/datasets/index`.
+
+.. tip::
+
+   **💡 When to modify test cases**
+   
+   * Review feedback indicates that test cases need adjustment (see :doc:`/hub/ui/annotate/review_test_results`)
+   * Test cases are not accurately representing the intended scenarios
+   * Checks need to be adjusted to better match evaluation criteria
+   * Test cases need to be organized with tags and descriptions
+   
+   This workflow is typically triggered after a business user reviews test results and identifies issues that need modification.
+
+Modify test cases
+^^^^^^^^^^^^^^^^^
+
+Draft/Undraft your test case
+-----------------------------
+
+Drafting and undrafting test cases allows you to control which test cases are included in evaluation runs.
+
+Setting a test case to draft status:
+
+* **Excludes it from evaluation runs** - Draft test cases are not used in evaluations until they are undrafted
+* **Indicates work in progress** - Shows that the test case is being reviewed or modified
+* **Prevents biased metrics** - Ensures that incomplete or problematic test cases don't affect your evaluation results
+
+To draft a test case:
+
+1. Open the test case (conversation) you want to draft
+2. Set it to draft status using the draft toggle or option
+3. The test case will be excluded from future evaluation runs until it is undrafted
+
+You can also set a test case to draft when creating a task from an evaluation run. This ensures that failed test cases are automatically excluded from subsequent evaluations until they are reviewed and fixed.
+
+.. tip::
+
+   For more information about creating tasks and setting test cases to draft, see :doc:`/hub/ui/annotate/task_management`.
+
+Hide/Unhide
+___________
+
+In addition to drafting, you can hide test cases to organize your evaluation overvieq:
+
+* **Hide** - Makes the test case less visible in the evaluation overview and for the metrics computations in the dashboard
+* **Unhide** - Makes the test case visible again in the evaluation overview
+
+.. tip::
+
+   You can look at understanding the overview of evaluations in :doc:`/hub/ui/evaluations/create`.
+
+Rerun the test case
+-------------------
+
+After modifying a test case or its checks, you should rerun the test to validate your changes.
+
+**When to rerun:**
+
+* After modifying the conversation structure
+* After updating the answer example
+* After enabling or disabling checks
+* After modifying check requirements
+* After making any changes that could affect the test result
+
+**How to rerun:**
+
+1. Make your modifications to the test case
+2. Use the "Rerun" or "Test" option to run the test case in isolation
```

Comment:
Keep it to "Test" (or test evaluation) here I think, since we are talking from the test case page